baseapp/baseapp.go

```diff
@@ -2,19 +2,17 @@ package baseapp
 
 import (
 	"context"
-	"errors"
 	"fmt"
 
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/log"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
-	dbm "github.com/tendermint/tm-db"
 
 	"github.com/cosmos/cosmos-sdk/codec/types"
+	dbm "github.com/cosmos/cosmos-sdk/db"
```

Comment:
We can't do this on master. It will break every app in production. We basically need a baseapp v2 for this rather than breaking the existing baseapp.

Reply:
Alright. I can look into splitting it out, but it may be tricky or infeasible to keep all the modules code compatible both ways.